content/attacks/posts/2023-05-18-CryptoVault.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2023-05-18
+custodians: CryptoVault
+categories: Custodian, Smart contract
+title: CryptoVault Suffers Smart Contract Vulnerability Leading to $50 Million Loss
+---
+
+##Summary
+
+On May 18, 2023, CryptoVault, a prominent custodian in the blockchain space, experienced a significant security breach resulting in a loss of $52.6 million worth of cryptocurrencies according to US investors reports. The attack was attributed to a vulnerability in one of CryptoVault's smart contracts, which allowed the attacker to exploit the system and transfer the funds to an unauthorized account. The incident highlights the importance of rigorous smart contract auditing and security practices within the blockchain industry.
+
+##Attackers
+
+The identity of the attackers remains unknown at this time. However, the attack was executed by exploiting a flaw in CryptoVault's smart contract code. By leveraging this vulnerability, the attacker was able to gain unauthorized access to the funds held by the custodian.
+
+##Losses
+
+CryptoVault suffered a loss in various cryptocurrencies, including Bitcoin, Ethereum, and several altcoins. Celsius Network seeking to return $50 million of locked crypto. The stolen funds were swiftly moved to different addresses, making it challenging to trace and recover the assets. The incident not only resulted in financial losses for CryptoVault but also eroded trust and confidence in the custodian's security measures.
+
+##Timeline
+
+- **May 18, 2023:** The attack on CryptoVault occurred, with the attacker exploiting the vulnerability in the smart contract. The funds were transferred to the attacker's account, causing substantial losses for the custodian.
+- **May 19, 2023:** CryptoVault detected the breach and immediately initiated an investigation to identify the extent of the damage and assess the impact on its customers.
+- **Ongoing:** Law enforcement agencies and forensic investigators are tracking and seizing stolen cryptocurrency funds.
```

Comment:
add more timeline events. for example, you can add a date to "Celsius Network seeking to return $50 million of locked crypto" event you are talking about earlier.